cdc/owner/feed_state_manager.go

```diff
@@ -416,12 +428,14 @@ func (m *feedStateManager) patchState(feedState model.FeedState) {
 	})
 }
 
-func (m *feedStateManager) cleanUpTaskPositions() {
```

Comment:
how it works?

Reply:
Cleanup checkpointTs and resolvedTs before stopping, and it will be initialized when changefeed resumes.